getting_started/workflow/assets/importing_scenes.rst

```diff
@@ -52,6 +53,24 @@ Godot provides a `Python
 Plugin <https://github.com/godotengine/collada-exporter>`__
 that will do a much better job of exporting the scenes.
 
+Exporting GLTF 2.0 files from Blender
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 ways to export GLTF files from Blender. As a GLTF binary (.glib file), GLTF embedded (.gltf file), 
+and with textures separate from the material (.gltf + .bin + textures).
+
+GLTF binary files are the smallest of the three options. It exports the mesh, and textures set up in Blender
+are part of the material file in Godot. If you don't need the texture files to be separate from the material
+file this is the format you should use.
+
+GLTF embedded files function the same way as binary files. They don't provide extra functionality in Godot,
+and shouldn't be used since they have a larger file size.
+
+GLTF with the textures separate from the material should be used if you need your textures to be separate from the
```

Comment:
That first sentence seems to state the obvious ;)

I'd mention that the main purpose of this distribution format is to have the `.glTF` scene description in a text-based format, friendly to version control, and the binary data in separate binary files.

Reply:
Could you be more specific on the advantage of having that information separate? I can't think of a scenario where I'd choose to have the data separate instead of just using a ``.glb`` file.

Reply:
I'm not familiar with actually using glTF so I can't speak for the real life benefits, but being able to read the scene file is useful for the same reason that Godot provides a text-based scene format. You can see changes made to the scene structure in `git diff`, revert partial changes or choose to commit only some chunks, and most importantly handle merge conflicts.